src/renderer/screens/manager/AppsList/AppIcon.js

```diff
@@ -26,13 +27,38 @@ const Container = styled.div`
   }
 `;
 
+const ManagerAppIconContainer = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  transition: opacity 0.2s ease-out;
+  color: ${p => p.theme.colors.palette.text.shade20};
+`;
+
+const IconContainer = styled.div`
+  width: ${p => p.size}px;
+  height: ${p => p.size}px;
+  position: relative;
+  ${ManagerAppIconContainer} {
+    opacity: ${p => (p.loading ? 1 : 0)};
+  }
+  > img {
+    transition: opacity 0.2s ease-out;
+    z-index: 1;
+    opacity: ${p => (p.loading ? 0 : 1)};
+  }
+`;
+
 type Props = {
   app: App,
   theme: any,
 };
 
 function AppIcon({ app, theme }: Props) {
   const { currencyId, icon } = app;
+  const [loading, setLoading] = useState(true);
+  const onLoad = useCallback(() => setLoading(false), []);
```

Comment:
What's the reason for using `useCallback` in this case?

Reply:
i kept it to not re-render the function in case of props changes